scripts/cloud-release.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+source ./scripts/cloud-utils.sh
+export RELEASE_ROLE_NAME=CodebuildRelease
```

Comment:
I think we need to create this console role and attach proper permission for it in the isengard

Reply:
As the e2e workflow also needs a console role, I think we can use one new role as `CodeBuild` or use the existing `Admin` role for all the build projects. Let's discuss before creating it